doc/modules/clustering.rst

```diff
@@ -1339,6 +1339,91 @@ mean of homogeneity and completeness**:
    <http://www.cs.columbia.edu/~hila/hila-thesis-distributed.pdf>`_, Hila
    Becker, PhD Thesis.
 
+.. _fowlkes_mallows_scores:
+
+Fowlkes-Mallows scores
+----------------------
+
+The Fowlkes-Mallows index (FMI) is defined as the geometric mean between of
+the precision and recal::
+
+      FMI = TP / sqrt((TP + FP) * (TP + FN))
+
+Where :math:`TP` is the number of `True Positive` (i.e. the number of pair
```

Comment:
I don't think those single backticks do anything. Do you want to emphasise? use asterisks.